binaries/README.md

```diff
@@ -99,4 +99,40 @@
    Note: `--nightly` option puts the *.whl files in a subfolder named 'nightly' in the specified bucket
    ```
    python s3_binary_upload.py --s3-bucket <s3_bucket> --s3-backup-bucket <s3_backup_bucket> --nightly
-   ```
\ No newline at end of file
+   ```
+
+## Uploading packages to production torchserve account
+### pypi
+Binaries should show up here: https://pypi.org/project/torchserve/
+
+You need to be on the list of maintainers to run the below.
+
+`twine upload <path/to.wheel>`
+
+
+### conda
+Binaries should show up here: https://anaconda.org/pytorch/torchserve
+
+
+```
+# Authenticate with pytorchbot credentials
+anaconda login
+
+# Upload binaries
+anaconda upload -u pytorch <path/to/.bz2>
+```
+
+## docker
+Binaries should show up here: https://hub.docker.com/r/pytorch/torchserve
+
+Change the staging org to your personal docker or test docker account https://github.com/pytorch/serve/blob/master/docker/promote-docker.sh#L8
+
+```
+# authenticate to docker with pytorchbot credentials
+docker login
+
+# upload docker image
+./docker/promote-docker.sh
+```
```

Comment:
To my understanding, eli@ requests us to provide the stage links of binary and docker. He only needs to tags them on production. Please confirm with eli@ to clarify.

Reply:
Yes that's correct so for docker this would mean you add the staging org `lxning` and for the binaries you can wget them from the staging links you create and then upload them with instructions from this doc

Let me clarify